main.go

```diff
@@ -44,13 +46,13 @@ func main() {
 		o.Development = true
 	}))
 
-	syncPeriod := 2 * time.Minute
+	syncPeriodD := time.Duration(int(time.Second) * syncPeriod)
```

Comment:
That might overflow on different machines.

I used the `36000` (10hours) to play with it. It was running okay on my glinux, but overflow in golang playground: https://play.golang.org/
```
package main

import (
	"fmt"
	"time"
)

func main() {
        var syncPeriod int = 36000
	syncPeriodD := time.Duration(int(time.Second) * syncPeriod)
	fmt.Println("syncPeriodD = ", syncPeriodD)
}
```

I got `syncPeriodD =  -415.875072ms`. I think we need to take care of it.

Reply:
I would think the playground is a 32 bit env.
explicitly using int64 works. 

func main() {
        var syncPeriod int64 = 36000
	syncPeriodD := time.Duration(int64(time.Second) * syncPeriod)
	fmt.Println("syncPeriodD = ", syncPeriodD)
}

Reply:
Fixed to use int64